blackbox/docs/general/builtins/scalar.rst

```diff
@@ -333,6 +333,26 @@ Examples
    +---------------------+
    SELECT 1 row in set (... sec)
 
+.. _scalar-quote-ident:
+
+``quote_ident(text)``
+---------------------
+
+Returns: ``text``
+
+Quotes a provided string argument. The string is quoted only when in is need.
+For example, if it contains non-identifier characters, keywords or would be
+case-folded. Embedded quotes are properly doubled.
+
+::
+
+   cr> select quote_ident('double precision');
```

Comment:
I think it would be clearer to use another example (not one with a data type) - this could be confusing